Backend/updateBackendEnv.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 pip install virtualenv
-virtualenv -p python2.6 env
```

Comment:
Taking this out breaks prod. Just something to be aware of. Might be better to add an additional script to make the prod virtual env temporarily then go back to one script when prod is fixed.